pkg/identity/cache/allocator.go

```diff
@@ -318,7 +318,13 @@ func (m *CachingIdentityAllocator) AllocateIdentity(ctx context.Context, lbls la
 	defer func() {
 		if err == nil {
 			if allocated || isNewLocally {
-				metrics.Identity.Inc()
+				if id.ID.HasLocalScope() {
+					metrics.Identity.WithLabelValues("cidr").Inc()
```

Comment:
Minor nit, I'd suggest we go with "node_local" nomenclature, it's more consistent with the "cluster_local", and will be more future-proof for other use of this allocation range.
```suggestion
					metrics.Identity.WithLabelValues("node_local").Inc()
```

Reply:
(also elsewhere; might even be something worth defining string constants for these labels somewhere so that we guarantee they're consistent between this usage and the other usages)

Reply:
I added a few string constants to represent each type and replaced the old usages.